spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/cloud_manager_spec.rb

```diff
@@ -0,0 +1,56 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager do
+  context ".raw_connect" do
+    let(:config) do
+      {
+        :provider               => "IbmCloud",
+        :IbmCloud_project         => "project",
+        :IbmCloud_json_key_string => "encrypted",
+        :app_name               => Vmdb::Appliance.PRODUCT_NAME,
+        :app_version            => Vmdb::Appliance.VERSION,
```

Comment:
We should go ahead and align the operators here like @agrare (and @miq-bot) have mentioned.

e.g.:
```
{
    :provider                 => "IbmCloud",
    :IbmCloud_project         => "project",
    :IbmCloud_json_key_string => "encrypted",
    :app_name                 => Vmdb::Appliance.PRODUCT_NAME,
    :app_version              => Vmdb::Appliance.VERSION,
}
```

Also try to fix up the other @miq-bot suggestions.